maxim/logger/livekit/instrumenter.py

```
@@ -74,4 +75,5 @@ def instrument_livekit(logger: Logger, callback: MaximLiveKitCallback = None):
             setattr(JobContext, name, instrument_job_context(orig, name))
 
     # Instrument gemini models if present
-    instrument_gemini()
\ No newline at end of file
+    instrument_gemini()
+    instrument_module_functions()
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)